examples/java8/src/main/java/org/apache/beam/examples/snippets/Snippets.java

```diff
@@ -0,0 +1,233 @@
+    }
+
+    {
+      // [START BigQueryReadQuery]
+      PCollection<Double> maxTemperatures = p
+          .apply(BigQueryIO.readTableRows().fromQuery(
+              "SELECT max_temperature FROM [clouddataflow-readonly:samples.weather_stations]"))
+          // Each row is of type TableRow
+          .apply(MapElements.into(TypeDescriptors.doubles()).via(
+              (TableRow row) -> (Double) row.get("max_temperature")));
+      // [END BigQueryReadQuery]
+    }
+
+    {
+      // [START BigQueryReadQueryStdSQL]
+      PCollection<Double> maxTemperatures = p
+          .apply(BigQueryIO.readTableRows().fromQuery(
+              "SELECT max_temperature FROM `clouddataflow-readonly.samples.weather_stations`")
+              .usingStandardSql())
+          // Each row is of type TableRow
+          .apply(MapElements.into(TypeDescriptors.doubles()).via(
+              (TableRow row) -> (Double) row.get("max_temperature")));
+      // [END BigQueryReadQueryStdSQL]
+    }
+
+    {
+      // [START BigQuerySchema]
+      String tableSchema = "source:STRING, quote:STRING";
+      // [END BigQuerySchema]
+    }
+
+    // [START BigQuerySchemaObject]
+    List<TableFieldSchema> fields = new ArrayList<>(Arrays.asList(
+        new TableFieldSchema().setName("source").setType("STRING").setMode("NULLABLE"),
+        new TableFieldSchema().setName("quote").setType("STRING").setMode("REQUIRED")));
+    TableSchema tableSchema = new TableSchema().setFields(fields);
+    // [END BigQuerySchemaObject]
+
+    {
+      String tableSpec = "clouddataflow-readonly:samples.weather_stations";
+      if (writeProject != "" && writeDataset != "" && writeTable != "") {
+        tableSpec = writeProject + ":" + writeDataset + "." + writeTable;
+      }
+
+      // [START BigQueryWrite]
+      /*
+      @DefaultCoder(AvroCoder.class)
+      static class Quote {
+        String source;
+        String quote;
+
+        public Quote() {}
+        public Quote(String source, String quote) {
+          this.source = source;
+          this.quote = quote;
+        }
+      }
+      */
+
+      PCollection<Quote> quotes = p.apply(Create.of(
+          new Quote("Mahatma Ghandi", "My life is my message.")
+      ));
+
+      quotes
+          .apply(MapElements.into(TypeDescriptor.of(TableRow.class)).via(
+              (Quote elem) -> {
+                TableRow row = new TableRow();
+                row.set("source", elem.source);
+                row.set("quote", elem.quote);
+                return row;
+              }
+          ))
+          .apply(BigQueryIO.writeTableRows()
+              .to(tableSpec)
+              .withSchema(tableSchema)
+              .withWriteDisposition(WriteDisposition.WRITE_TRUNCATE)
+              .withCreateDisposition(CreateDisposition.CREATE_IF_NEEDED));
+      // [END BigQueryWrite]
+    }
+  }
+
+  /** Helper function to format results in coGroupByKeyTuple. */
+  public static String formatCoGbkResults(
```

Comment:
Is this related to bigquery snippets?

Reply:
No, I renamed the directory and git apparently took it as replacing it with a new file. This part is unchanged.